### PR TITLE
Rework Wallet Page

### PR DESCRIPTION
**Pull request by jvonasek**

### CODE REVIEW

File: src/sections/trade/TradePage.tsx [hunk 1]
```diff
@@ -1,18 +1,67 @@
...
+const isDcaPageEnabled = import.meta.env.VITE_FF_DCA_ENABLED === "true"
+const isBondsPageEnabled = import.meta.env.VITE_FF_BONDS_ENABLED === "true"
 
 export const TradePage = () => {
   const { t } = useTranslation()
   const matchRoute = useMatchRoute()
   const isBondPage = matchRoute({ to: LINKS.bond })
   const isDesktop = useMedia(theme.viewport.gte.sm)
 
+  const subNavigationItems = useMemo(() => {
```
Comment (vkulinich-cl) [inline src/sections/trade/TradePage.tsx:28]: Is there away to move it out from component above and use it as array?

File: src/sections/trade/BondsSubNavigationTabLink.tsx [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+      })
+    }
+  }, [bonds, currentBlockNumber, lbpPool.data])
+
+  return (
+    <SubNavigationTabLink
+      to={LINKS.bonds}
+      icon={<IconBonds />}
+      label={t("header.trade.bonds.title")}
+      badge={isActive ? t("header.trade.sale") : ""}
```
Comment (vkulinich-cl) [inline src/sections/trade/BondsSubNavigationTabLink.tsx:45]: it should be saying Active, not Sale

Comment (jvonasek) [reply src/sections/trade/BondsSubNavigationTabLink.tsx]: fixed, sorry

**Review by vkulinich-cl — APPROVED** [review]

**Comment by jak-pan** [comment]
There is an issue with medium size screens not showing full buttons

**Comment by vkulinich-cl** [comment]
> There is an issue with medium size screens not showing full buttons

it was made by purpose, because all buttons is not fitting the table width

**Comment by jak-pan** [comment]
> > There is an issue with medium size screens not showing full buttons
> 
> it was made by purpose, because all buttons is not fitting the table width

That doesn't look like on purpose 

Screenshot 2023-12-08 at 00 00 18

**Pull request merged**